GLAA.Web/Views/OrganisationDetails/CommunicationPreference.cshtml

```diff
@@ -4,9 +4,9 @@
 
 @model GLAA.ViewModels.LicenceApplication.CommunicationPreferenceViewModel
```

Comment:
These file renames are going to save a lot of time trying to figure out which fields are on which pages!
+10